### PR TITLE
ci: update cache dependencies to actions/cache@v2

### PR DESCRIPTION
**Pull request by viestat**
This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Tests for the changes have been added if needed.
- [x] Does not affect documentation or it has been added or updated.

### CODE REVIEW

File: .github/workflows/continuous-integration.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
         with:
           node-version: 12
       - name: Cache dependencies
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           key: npm-${{ hashFiles( 'package-lock.json') }}
```
Comment (JrSchild) [inline .github/workflows/continuous-integration.yaml:27]: Nope

**Review by JrSchild — CHANGES_REQUESTED** [review]

File: .github/workflows/continuous-integration.yaml (hunk 1 above)
Comment (viestat) [reply .github/workflows/continuous-integration.yaml]: fixed in https://github.com/ridedott/back-end-assignment-boilerplate/pull/192/commits/d2e9be3d7774ca5e6e8caf843842a0961aaa0c82

File: .github/workflows/continuous-integration.yaml [hunk 2]
```diff
@@ -22,9 +22,9 @@ jobs:
         with:
           node-version: 12
       - name: Cache dependencies
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
-          key: npm-${{ hashFiles( 'package-lock.json') }}
+          key: npm-${{ hashFiles( './package-lock.json') }}
```
Comment (JrSchild) [inline .github/workflows/continuous-integration.yaml:27]:
```suggestion
          key: npm-${{ hashFiles('./package-lock.json') }}
```

**Review by JrSchild — CHANGES_REQUESTED** [review]

**Review by JrSchild — APPROVED** [review]

File: .github/workflows/continuous-integration.yaml [hunk 3]
```diff
@@ -46,9 +46,9 @@ jobs:
...
```
Comment (JrSchild) [inline .github/workflows/continuous-integration.yaml:51]: Come on now. 
```suggestion
          key: npm-${{ hashFiles('./package-lock.json') }}
```

**Review by JrSchild — CHANGES_REQUESTED** [review]

**Review by JrSchild — APPROVED** [review]

**Pull request merged**